src/administration/horizontal_scaling.md

```diff
@@ -0,0 +1,141 @@
+# Scaling Lemmy horizontally
+
+This is a collection of notes on scaling different Lemmy components horizontally. This is not meant as a step-by-step
+guide, rather as general tips and knowledge that might be useful to somebody who is already familiar with and horizontal scaling concepts and best
+practices. If you don't already know about concepts like load balancing,
+object storage, and reverse proxies, then this document will probably be hard (or impossible) to understand.
+
+**Note: scaling Lemmy this way is not necessary, and for small instances, the added overhead most likely
+outweighs any benefits.**
+
+## Why scale horizontally?
+
+- If one of your Lemmy servers dies for any reason, your instance can still remain operational
+- Having multiple Lemmy servers allows you to do rolling upgrades (no downtime needed to upgrade Lemmy!)
```

Comment:
Maybe add a qualifier here about that wmight cause really weird or dangerous issues if the new version has larger database migrations ?

edit: i see you have info about it further below